src/states_screens/credits.cpp

```diff
@@ -68,6 +68,12 @@ class CreditsSection
     // ------------------------------------------------------------------------
     void addSubEntry(stringw& subEntryString)
     {
+        //Add a blank entry if no entry exists for this sub entry.
+        if(m_entries.size() == 0){
```

Comment:
Can we have the '{' on a new line? And also a space after the 'if'.
See section about conditionals https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Conditionals.